### ISSUE TITLE
Server-side viewer panels: adjustments to layout and functionality

### ISSUE THREAD

**Issue by clarasb** [xcube-dev/xcube-viewer#452]
This issue holds ideas to enhance the layout and functionalty of server-side panels in the viewer.

General idea:
- When creating new places, the viewer automatically jumps to the 'Time Series' panel. This is a bit confusing if the user is already working in a panel and wants to create a new place to work with in that panel.

Ideas related to the demo panel '2D-Histogramm':
- Usually, the control features (buttons etc.) are placed at the top of the panel. For consistency, the controls in this panel could also be placed at the top instead of the bottom.
- Adding the place name to the plot, otherwise there is no obvious/visual connection between the plot in the panel and the place.

![Screenshot 2024-11-29 150238](https://github.com/user-attachments/assets/9b452b97-d843-4c2a-9aea-b81cb8027009)

**Comment by forman** [comment]
> - When creating new places, the viewer automatically jumps to the 'Time Series' panel. This is a bit confusing if the user is already working in a panel and wants to create a new place to work with in that panel.

Yes, this is the default behaviour. You can switch it off in the settings. 

> - Usually, the control features (buttons etc.) are placed at the top of the panel. For consistency, the controls in this panel could also be placed at the top instead of the bottom.

You are right, will do. 

> - Adding the place name to the plot, otherwise there is no obvious/visual connection between the plot in the panel and the place.

Again, right, will do.
